.github/workflows/keep-workflows-enabled.yaml

```diff
@@ -38,15 +39,21 @@ jobs:
           - { repo: ncov,            workflow: rebuild-100k.yml }
           - { repo: ncov-ingest,     workflow: fetch-and-ingest-genbank-master.yml }
           - { repo: ncov-ingest,     workflow: fetch-and-ingest-gisaid-master.yml }
+          - { repo: nextclade_data,  workflow: update-sars-cov-2-datasets.yml }
```

Comment:
cc:@ivan-aksamentov, just so you are aware that we use this GH Action workflow to keep other scheduled workflows enabled.